sdk/storage/azure-storage-blob/azure/storage/blob/aio/blob_client_async.py

```diff
@@ -138,11 +136,8 @@ def __init__(
     async def upload_blob(
             self, data,  # type: Union[Iterable[AnyStr], IO[AnyStr]]
             blob_type=BlobType.BlockBlob,  # type: Union[str, BlobType]
-            overwrite=False,  # type: bool
```

Comment:
Please open an issue to change the :param: directive for the keyword parameter to :keyword: in the docstring.

Reply:
https://github.com/Azure/azure-sdk-for-python/issues/7619